DigitalLearningSolutions.Web/Controllers/TrackingSystem/CentreConfiguration/RegistrationPromptsController.cs

```diff
@@ -302,6 +388,31 @@ private IActionResult AddRegistrationPromptConfigureAnswersPostNext(Registration
             return RedirectToAction("AddRegistrationPromptSummary");
         }
 
+        private IActionResult AddRegistrationPromptBulk(RegistrationPromptAnswersViewModel model)
+        {
+            UpdateTempDataWithAnswersModelValues(model);
+            return RedirectToAction("AddRegistrationPromptBulk");
+        }
+
+        private IActionResult EditRegistrationPromptBulk(EditRegistrationPromptViewModel model)
+        {
+            var data = new EditRegistrationPromptData();
+            var id = data.Id;
+            data.EditModel = model;
+
+            Response.Cookies.Append(
+                EditCookieName,
+                id.ToString(),
+                new CookieOptions
+                {
+                    Expires = DateTimeOffset.UtcNow.AddDays(1)
+                }
+            );
+            TempData.Set(data);

```

Comment:
Nice - two questions about this.
1. Could we pass "model" into the EditRegistrationPromptData constructor and set the EditModel when creating the object?
2. Could we pull the L399-411 into a private method "SetEditRegistrationPromptTempData" or similar?